compiler/damlc/lib/DA/Cli/Visual.hs

```diff
@@ -125,14 +127,17 @@ constructSubgraphsWithLables lookupData tpla@TemplateChoices {..} = SubGraph nod
         nodes = map (\(chc, _) -> (chc, nodeIdForChoice lookupData chc)) choicesInTemplete
         nodesWithCreate = nodes ++ [addCreateChoice tpla lookupData]
 
-actionToChoice :: LF.Template -> Action -> LF.ChoiceName
-actionToChoice tpl (ACreate _) = LF.ChoiceName $ tplName tpl <> "_Create"
-actionToChoice tpl (AExercise _ (LF.ChoiceName "Archive")) = LF.ChoiceName $ tplName tpl <> "_Archive"
-actionToChoice _tpl (AExercise _ chc) = chc
+tplNamet :: LF.TypeConName -> T.Text
```

Comment:
You're really unqualifying the name right? So maybe `tplNameUnqual`?

Reply:
that sounds a lot better. Done